csvcubed/csvcubed/utils/qb/standardise.py

```diff
@@ -16,6 +17,39 @@
     NewQbMeasure,
     QbMultiMeasureDimension,
 )
+from csvcubed.utils.uri import uri_safe
+
+
+def standardise_categoricals(data: pd.Series) -> pd.Series:
+    """Standardise categorical data assuming case insensitivity along highest sorted() instance of uri_safe result"""
+    if data.cat.categories.dtype != "O":
```

Comment:
What insanity is this? I can only hope `O` stands for `Ordinal`.

Reply:
O stands for Object. Don't blame me, blame Pandas.